deployments/dev/environment.py

```diff
@@ -33,13 +34,18 @@ def env() -> Mapping[str, Optional[str]]:
 
         'AZUL_DEPLOYMENT_STAGE': 'dev',
 
-        'AZUL_CATALOGS': ','.join([
-            'hca:dcp2:repository/tdr:metadata/hca',
-            'hca:dcp2ebi:repository/tdr:metadata/hca',
-            'lungmap:lungmap:repository/tdr:metadata/hca',
-            'hca:it2:repository/tdr:metadata/hca',
-            'hca:it2ebi:repository/tdr:metadata/hca',
-            'lungmap:it3lungmap:repository/tdr:metadata/hca'
+        'AZUL_CATALOGS': json.dumps([
+            dict(name=name,
```

Comment:
We should exploit/enforce that catalog names are unique. This structure should be a dictionary instead of a list where the keys are the catalog name.

Reply:
@hannes-ucsc during parking lot you requested that the JSON structure matches the response from the `index/catalogs` endpoint.

The plugin format within the response looks like
```JSON
      "plugins": [
        {
          "type": "repository",
          "name": "tdr",
          "sources": [
            "tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210524_lattice:"
          ]
        },
        {
          "type": "metadata",
          "name": "hca"
        }
      ]
```

The `cls.Catalog` on `develop` expects the plugins attribute to be a dictionary, and performs inspections in various places to `Catalog.plugins['plugin_type']`. I can refactor these inspections, but first wanted to ask if we want to change the format of `plugins` in the response to be a dictionary: 

```JSON
"plugins": {
   "repository": {"name":"tdr", "sources": [...]},
   "metadata":{"name": "hca"}
}
```

I asked the data-browser team, and they do not use the `plugins` attribute, so if we decide to change it there should be no need to coordinate a deployment.

Reply:
Changing the `/index/response` from list-of-dicts to dict-of-dicts is probably better. I honestly don't remember why I modeled this as list-of-dicts. I don't think we'll ever have multiple plugins of the same type per catalog.